account.go

```diff
@@ -118,13 +117,13 @@ func (p *AccountExternalAccountParams) AppendTo(body *form.Values, keyParts []st
 type PayoutScheduleParams struct {
 	DelayDays     uint64   `form:"delay_days"`
 	Interval      Interval `form:"interval"`
-	MinimumDelay  bool     `form:"-"` // See custom AppendTo
+	MinimumDelay  *bool    `form:"-"` // See custom AppendTo
```

Comment:
Off topic, but should we rename this field? It might be better called `DelayDaysMinimum` or such so that it gets alphabetized with `DelayDays`.

Reply:
ah yes good idea, I can take that one

Reply:
fixed!